### DISCUSSION TITLE
Is TRENDY v10  or CRUJRA v2.2 forcing available in FATES friendly format?

### DISCUSSION THREAD

**Discussion by JessicaNeedham** [NGEET/fates#957] [Q&A]
For the demographic  benchmarking model inter-comparison  project we need to use TRENDY v10  forcing  based on CRU-JRA 2.2 (1901-2020).  Does anyone know if these versions are available somewhere in a format that FATES can read - one file per month for  Solar, Precip and TPQWL? 

Both compy and cori have CRUJRA  but I'm not sure if it is the correct version. On cori CRUJRA forcing is here: /global/cfs/cdirs/e3sm/inputdata/atm/datm7/atm_forcing.CRUJRA/ 

Both compy and  cori  data have the following met data:
global attributes : CRUNCEP 6-Hourly Atmospheric Forcing: Incoming Solar.  Missing values filled with 1948 Qian data interpolated by datm to CRUNCEP grid.  See CRUNCEP_Landmask.nc in /glade/p/cesm/lmwg/atm_forcing.datm7.cruncep_qianFill.0.5d.V4.c130305 for valid (not filled) CRUNCEP data.

I don't see a version number. Anyone have any idea  if that is v2.2? 

Thanks!

**Comment by ckoven** [comment]
Looks like some relevant discussion in https://github.com/ESCOMP/CTSM/issues/1895?

**Reply by wwieder** [reply]
Yes, Daniel has the data on campaign store (Casper)
/glade/campaign/asp/djk2120/TRENDY2022/forcing

Let me know if you need machine access here, Jessie.
Otherwise, Daniel may need to adjust permissions to the files for you to read them?

**Reply by JessicaNeedham** [reply]
Great, thanks Charlie and Will. I can see the relevant forcing on Casper. File permissions are rwxr-xr-x so I think I should be able to copy them to cori or compy. Does Casper  have a globus endpoint?

**Comment by olyson** [comment]
I recently created a Globus guest collection for this.  Try this link:

https://app.globus.org/file-manager?origin_id=15ed38d8-58dd-44a8-9b87-cdf7b24f82dc&origin_path=%2F

Since this dataset includes JRA-55 data, acknowledgment should include the following: "The dataset used for this study includes data from the Japanese 55-year Reanalysis (JRA-55) project carried out by the Japan Meteorological Agency (JMA)." Furthermore, note that "Individual users should provide JMA ([jra@met.kishou.go.jp](mailto:jra@met.kishou.go.jp)) with a copy of their scientific or technical papers, publications, press releases or other communications regarding these datasets."

**Reply by JessicaNeedham** [reply]
Thanks @olyson ! The link is working. I'm copying over to cori here:  /global/cfs/cdirs/e3sm/inputdata/atm/datm7/TRENDY_2022  although I'll probably have to move it to perlmutter soon. 

Thank you for the acknowledgements  - I will forward to the leads on the demographic benchmarking project.
